app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Vaccine must be")

```

Comment:
The error message for `NotVaccinatedError` should match the exact wording specified in the checklist. Please update it to: `Visitor must have a vaccine`.